README.md

```diff
@@ -26,6 +26,7 @@
 - [CompetitiveCode](https://github.com/Vishruth-S/CompetitiveCode)
 - [DevJoke](https://github.com/shrutikapoor08/devjoke)
 - [Developer Experience Knowledge Base](https://github.com/DXHeroes/knowledge-base-content)
+- [Grid-Garden] (https://github.com/thomaspark/gridgarden/)
```

Comment:
Can you remove the blank space between the name of repository and the link?

Reply:
sure

Reply:
done! :D